.github/workflows/codeql-analysis.yml

```diff
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "release-1.15" ]
```

Comment:
Maybe instead of having different files per branch listing only the branch we are on, we could use a pattern like `release-*` and keep `main` in the list? Then we could take decision like what channel to push to based on the name of the branch.

Reply:
Brilliant, thanks.
Done in #424 please take a look

Reply:
The same change added here too for testing purposes, but when #424 is merged there will be a rebase here

Reply:
Rebased